pipenv/environments.py

```diff
@@ -35,3 +35,8 @@
 # Disable spinner on windows.
 if os.name == 'nt':
     PIPENV_NOSPIN = True
+
+_pipenv_timout_raw = os.environ.get('PIPENV_TIMEOUT')
```

Comment:
Perhaps we could just set 120 as the fallback here and save the extra lines?

`int(os.environ.get('PIPENV_TIMEOUT', 120)`